drivers/filesystems/fastfat/string.c

```diff
@@ -24,3 +25,10 @@ vfatIsLongIllegal(
 {
     return wcschr(long_illegals, c) ? TRUE : FALSE;
 }
+
+BOOLEAN
+IsDotOrDotDot(PCUNICODE_STRING File)
```

Comment:
I would prefer the name "FileName" instead of just "File", but it's just my own opinion. Otherwise good to go!

Reply:
How about just "Name" since this could be a File Name or a Directory Name? If you agree, then "fixed". Thanks.